src/protocols/ssh/settings.c

```diff
@@ -279,6 +287,14 @@ guac_ssh_settings* guac_ssh_parse_args(guac_user* user,
         guac_user_parse_args_boolean(user, GUAC_SSH_CLIENT_ARGS, argv,
                 IDX_CREATE_RECORDING_PATH, false);
 
+    /* Parse server alive interval */
+    settings->server_alive_interval =
+        guac_user_parse_args_int(user, GUAC_SSH_CLIENT_ARGS, argv,
+                IDX_SERVER_ALIVE_INTERVAL, 0);
+    if (settings->server_alive_interval == 1)
+        guac_user_log(user, GUAC_LOG_WARNING, "Minimum keepalive interval "
```

Comment:
Perhaps this should instead be logged within the shared SSH client code, rather than repeated for all users of that shared code?